src/Consolonia.Core/ConsoloniaLifetime.cs

```diff
@@ -0,0 +1,217 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.ApplicationLifetimes;
+using Avalonia.Platform;
+using Avalonia.Threading;
+using Consolonia.Core.Drawing.PixelBufferImplementation;
+using Consolonia.Core.Infrastructure;
+
+// ReSharper disable CheckNamespace
+// ReSharper disable NotNullOrRequiredMemberIsNotInitialized
+// ReSharper disable ConstantConditionalAccessQualifier
+namespace Consolonia
+{
+    public class ConsoloniaLifetime : ISingleViewApplicationLifetime,
```

Comment:
May be we should propose Avalonia to split their implementation of single view lifetime into core packages just to re-use that among platforms. Sounds meaningful?